internal/cli/command/root.go

```diff
@@ -14,13 +13,13 @@ var cfgFile string
 
 var rootCmd = &cobra.Command{
 	Use:   "OCTAVIUS",
-	Short: "Welcome to the ocatvius cli",
-	Long:  `Easily automate your work using ocatvius' multi-processing capabilities`,
+	Short: "Welcome to the octavius cli",
+	Long:  `Easily automate your work using octavius' multi-processing capabilities`,
 }
 
 func Execute() {
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Println(err)
+		logger.Error("Execution error ", err)
```

Comment:
Error strings should not be capitalized. [ref](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)